contracts/ts/utils.ts

```diff
@@ -101,7 +99,11 @@ export const getDefaultNetwork = async (): Promise<Network | undefined> => {
  * @dev to be used while testing
  * @returns the signers
  */
-export const getSigners = async (): Promise<Signer[]> => ethers.getSigners();
+export const getSigners = async (): Promise<Signer[]> => {
+  const { ethers } = await import("hardhat");
```

Comment:
I'm curious why we do this? is it because of typedoc complaining? and why is it `await import` and not just `import`

Reply:
This is because of hardhat. Importing this is not available during hardhat bootstrap so that's why I use dynamic import (await is needed here).

Reply:
https://github.com/privacy-scaling-explorations/maci/pull/1106#discussion_r1467791954